### ISSUE TITLE
Enable alt text in PFPs and Profile Banners

### ISSUE THREAD

**Issue by jstewart612** [bluesky-social/social-app#6423]
### Describe the Feature

Enable alt text in PFPs and Profile Banners.

**Comment by dworthyn** [comment]
Seconded. Alt Text is needed for all images on Bluesky

**Comment by AstraTheDragon** [comment]
Agreed! The goal always should be to set new, higher, better standards of accessibility

**Comment by claromes** [comment]
Duplicate https://github.com/bluesky-social/social-app/issues/2252
